website/src/css/custom.css

```diff
@@ -181,6 +181,3 @@ html[data-theme='dark'] img[src$='#gh-light-mode-only'] {
 .test-marker-site-custom-css-unique-rule {
   content: "site-custom-css-unique-rule";
 }
-.test-marker-site-custom-css-shared-rule {
-  max-width: 100%;
-}
```

Comment:
This test is removed. We can't guarantee what this rule will be merged into because other bundled CSS may change. We can only guarantee site CSS is inserted after client module CSS.